packages/type-eval/src/print/json.ts

```diff
@@ -38,7 +38,11 @@ export function schemaToJson(
 
   switch (schema.kind) {
     case "primitive":
-      return { type: schema.name };
+      return {
+        type: schema.name,
+        documentationComment: schema.node?.symbol?.getDocumentationComment?.() ?? [],
+        jsDocTags: schema.node?.symbol?.getJsDocTags?.() ?? [],
+      };
```

Comment:
wanted to do this for other schemas as well, but this only works on primitive. For other schemas, `getDocumentationComment` and `getJsDocTags` always return an empty array... Need to look more into it